### PR TITLE
systemtests - build olm operator registry from olm-manifests

### PR DESCRIPTION
**Pull request by famartinrh**
### Type of change

- Enhancement / new feature

### Description

Comes from #4389 

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [x] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by kornys** [comment]
from log of GH I see it tries to push from container to localhost:500, I think you must provide IP of minikube node.

**Comment by famartinrh** [comment]
@enmasse-ci run tests OLM profile=acceptance

**Comment by famartinrh** [comment]
@enmasse-ci run tests OLM profile=acceptance test_only

**Comment by famartinrh** [comment]
@enmasse-ci run tests OLM profile=acceptance test_only

**Comment by famartinrh** [comment]
@enmasse-ci run tests OLM profile=acceptance test_only

File: .github/scripts/olm_test.sh [hunk 1]
```diff
@@ -4,5 +4,7 @@ set -e
 TAG=${TAG:-latest}
 export TEMPLATES=${PWD}/templates/build/enmasse-${TAG}
 
+export OCP4_EXTERNAL_IMAGE_REGISTRY=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' image-registry):5000
```
Comment (lulf) [inline .github/scripts/olm_test.sh:7]: Given that this is only run on minikube, does it need to be set? Or should we name it CLUSTER_EXTERNAL_IMAGE_REGISTRY?

Comment (famartinrh) [reply .github/scripts/olm_test.sh:7]: yes yes it need to be set, it makes the trick so the push to image registry works.
Rename it to CLUSTER_EXTERNAL_IMAGE_REGISTRY could make sense

**Comment by famartinrh** [comment]
@enmasse-ci run tests OLM profile=acceptance test_only

**Comment by famartinrh** [comment]
@enmasse-ci run tests OLM profile=acceptance test_only

**Comment by famartinrh** [comment]
@enmasse-ci run tests OLM profile=acceptance test_only

**Comment by famartinrh** [comment]
@enmasse-ci run tests OLM profile=acceptance test_only

**Comment by famartinrh** [comment]
@enmasse-ci run tests OLM profile=acceptance test_only

**Pull request merged**